README.md

```diff
@@ -66,10 +66,9 @@ def lambda_handler(event, context):
     corva.stream(stream_app, event)
 ```
 
-`Corva.stream` provides two optional parameters:
-
-- `filter_by_timestamp` - enable to clear [event](#event) from data with previously processed `timestamp`;
-- `filter_by_depth` - enable to clear [event](#event) from data with previously processed `measured_depth`.
+`Corva.stream` provides an optional parameter:
+- `filter_mode` - set to `timestamp` or `depth` to clear [event](#event) 
```

Comment:
Can you sync this with Ivan so that this gets ported over to Node SDK as well?

Reply:
Will sync